core/scripts/chaincli/handler/keeper_deployer.go

```diff
@@ -19,7 +19,7 @@ import (
 
 	"github.com/smartcontractkit/chainlink/core/scripts/chaincli/config"
 
-	offchain "github.com/smartcontractkit/ocr2keepers/pkg/config"
+	offchain20config "github.com/smartcontractkit/ocr2keepers/pkg/v2/config"
```

Comment:
🏄 A shorter alias would be preferable, like maybe `v2config`/`configv2`?

Reply:
yep, this is just keepers file so will update

Reply:
discussed offline, will keep as is to have consistency with delegate.go